app/controllers/race_editions_controller.rb

```diff
@@ -6,15 +8,15 @@ def index
   end
 
   def show
-    # binding.pry
+    race_edition = RaceEdition.where(id: @race_edition).includes(:race, race_entries: :racer).first
```

Comment:
This line runs a second SQL query to find the RaceEdition record and include its various relations (race, race_entries, and racers). This avoids getting N + 1 queries when manipulating the data in the presenter and the view.